polkadot/node/core/pvf/src/prepare/worker_intf.rs

```diff
@@ -108,25 +109,17 @@ pub async fn start_work(
 	metrics: &Metrics,
 	worker: IdleWorker,
 	pvf: PvfPrepData,
-	artifact_path: PathBuf,
+	cache_path: PathBuf,
 ) -> Outcome {
 	let IdleWorker { stream, pid, worker_dir } = worker;
 
-	gum::debug!(
-		target: LOG_TARGET,
-		worker_pid = %pid,
-		?worker_dir,
-		"starting prepare for {}",
-		artifact_path.display(),
-	);
-
```

Comment:
Would be nice to keep this. Maybe we can print the `pvf` if that's not too noisy, or print something else to identify the artifact. This log has been useful to me in the past.